src/EventGrid/EventGrid.Test/SessionRecords/Microsoft.Azure.Commands.EventGrid.Tests.ScenarioTests.DomainTests/EventGrid_DomainsGetKey.json

```diff
@@ -378,7 +378,7 @@
           "-1"
         ]
       },
-      "ResponseBody": "{\r\n  \"key1\": \"04c2gmm9FgEzK1Y7CsC28OgkPTaFv0Lb/nVm+JWcSHM=\",\r\n  \"key2\": \"KQ91yZDP1hLFlB1K1RJkdw87rt97Js0GKcg7fWu4A94=\"\r\n}",
+      "ResponseBody": "{\r\n  \"key1\": \"<HIDDEN>\",\r\n  \"key2\": \"<HIDDEN>\"\r\n}",
```

Comment:
So, if these are actual secrets that persist in Azure after this test, then you need to report their discosure on https://reportitnow

If these are just credentials that are used during the test and do not persist, you can just add a CredScan Suppression

Reply:
Thanks for the info @markcowl .. These don't get persisted after the tests as we delete these topics/domains and all their SAS keys are gone. We remove them to please the static analysis but we will address suppression next time.